src/main/java/uk/gov/justice/hmpps/prison/security/AuthenticationFacade.java

```diff
@@ -55,6 +57,7 @@ public static boolean hasRoles(final String... allowedRoles) {
     }
 
     public boolean isOverrideRole(final String... overrideRoles) {
+        if (overrideRoles.length > 0) log.error("Matching role check failed - no roles to match against");
```

Comment:
This will be removed in a later PR

Reply:
I don't understand - the if says there are roles but the error says there aren't

Reply:
yes - that is wrong